### PR TITLE
test/multi_stress_watch: use internel API to connect cluster

### PR DESCRIPTION
**Pull request by changchengx**
use connect_cluster_pp API to connect cluster

Signed-off-by: Changcheng Liu <changcheng.liu@aliyun.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by changchengx** [comment]
@tchaikov Please help review.

**Comment by changchengx** [comment]
@tchaikov Please help review.

**Comment by changchengx** [comment]
jenkins, test this please

**Comment by changchengx** [comment]
jenkins, test this please

**Comment by tchaikov** [comment]
> @tchaikov Please help review.

@changchengx will do. sorry for the latency.

**Comment by changchengx** [comment]
@tchaikov No problem. Thanks for your time.
I submit kinds of PR when reading the code and fix (potential) problems or just improve the readability of code to make it easy to be understood.

File: src/test/multi_stress_watch.cc [hunk 1]
```diff
@@ -170,6 +170,7 @@ int main(int args, char **argv)
   else if (type == "ec")
     test_erasure(cluster, pool_name, obj_name);
 
+  cluster.shutdown();
```
Comment (tchaikov) [inline src/test/multi_stress_watch.cc:156]: `Rados` 's destructor calls `Rados::shutdown()`. so this is not necessary.

File: src/test/multi_stress_watch.cc [hunk 2]
```diff
@@ -141,30 +141,13 @@ int main(int args, char **argv)
...
-  ret = cluster.connect();
-  if (ret) {
-    std::cerr << "Error " << ret << " in cluster.connect" << std::endl;
-    return ret;
+  std::string err = connect_cluster_pp(cluster);
+  if (err.length()) {
+      std::cerr << "Error " << err << std::endl;
+      return 1;
   }
+
```
Comment (tchaikov) [inline src/test/multi_stress_watch.cc:150]: thanks.

File: src/test/multi_stress_watch.cc (hunk 1 above)
Comment (changchengx) [reply src/test/multi_stress_watch.cc]: Thanks for reminding.

**Review by tchaikov — APPROVED** [review]

**Comment by tchaikov** [comment]
https://pulpito.ceph.com/kchai-2020-08-29_13:54:19-rados-wip-kefu-testing-2020-08-29-1945-distro-basic-smithi/

**Pull request merged**